sdk/appconfiguration/Azure.Data.AppConfiguration/src/AzureClientBuilderExtensions.cs

```diff
@@ -2,8 +2,11 @@
 // Licensed under the MIT License.
 
 using Azure.Core.Extensions;
+using Azure.Data.AppConfiguration;
 
-namespace Azure.Data.AppConfiguration
+#pragma warning disable AZC0001 // namespace
```

Comment:
Nit: different comment than the others (the others are more descriptive as to "why").